animal/animal.py

```diff
@@ -0,0 +1,17 @@
+class Animal:
```

Comment:
There is no need to have this class defined in separate file. Move this class definition to main.py.